### PR TITLE
Update " mkasset"

### PR DESCRIPTION
**Pull request by LegacyID1991**
Add so that mb bat doesn't give error when new asset is created and folders are empty.

### CODE REVIEW

File: bin/_mkproject.bat [hunk 1]
```diff
@@ -24,8 +24,8 @@ if "%2"=="" goto :missing_name
 if "%3"=="" goto :missing_silo
 
 set PROJECTDIR=%1%2
-set curPROJECT=%1
-set curPROJECTNAME=%2
+set CURRENTPROJECT=%1
```
Comment (mottosso) [inline bin/_mkproject.bat]: Excellent! Would it be possible to add a comment about what these are for? For our future selves, and other developers coming along.

File: bin/_mkproject.bat [hunk 2]
```diff
@@ -24,8 +24,8 @@ if "%2"=="" goto :missing_name
...
+set CURRENTPROJECT_NAME=%2
 set PROJECTSTAGE=%3
```
Comment (mottosso) [inline bin/_mkproject.bat]: This variable is also available as `MINDBENDER_SILO` below, this can safely be removed.

Comment (LegacyID1991) [inline bin/_mkproject.bat]: and it will when back2.0 or any other solution has been created

File: bin/_mkproject.bat (hunk 1 above)
Comment (LegacyID1991) [inline bin/_mkproject.bat]: definitely. I am on it

**Pull request closed without merging**